assets/javascripts/discourse/initializers/d-templates.js

```diff
@@ -79,12 +80,9 @@ function addKeyboardShortcut(api, container) {
             textarea: activeElement,
           });
         } else {
-          const modalController = container.lookup(
-            "controller:discourse-templates-modal"
-          );
-
-          showModal("discourse-templates-modal");
-          modalController.set("textarea", activeElement);
+          getOwner(this)
```

Comment:
I don't know what `this` is in this context. It's probably better to use the available `container` to do the `lookup()`

Reply:
Done